PG client customization (#1112)
Scalingo/documentation: _posts/databases/postgresql/2000-01-01-start.md

{% include mdl_img.html %}

PostgreSQL tends to use all the available memory if there is enough indices to fill the memory. If there is too many indices to fit into the memory, some of them are stored on the disk. In this situation, queries needing these indices will be slowed down. PostgreSQL first needs to load the indices from the disk into the RAM which takes some time. The memory usage on the "Metrics" tab of your web dashboard would always be at 100% in such situation.

> ## Client customization 

Reviewer: ```suggestion
## Client Customization 
```

> You can customize the PG client via addind a `.psqlrc` in your app directory

Reviewer: ```suggestion
You can customize the PostgreSQL client by adding a `.psqlrc` in your app directory.
```
